queries/ruby/highlights.scm

```diff
@@ -70,11 +73,6 @@
             ] @function
    )
 
-(program
- (call
-  (identifier) @include)
- (#vim-match? @include "^(require|require_relative|load)$"))
-
```

Comment:
There was a discussion whether not-top-level requires should be include: https://github.com/nvim-treesitter/nvim-treesitter/issues/463

@bluz71

Reply:
@theHamsta, Thanks for pointing me to that. I changed to be a `method_call`, it's looking like that in my machine?

<img width="400" alt="_HOME_src_ruby-highlight_rb" src="https://user-images.githubusercontent.com/120483/102671860-a506aa80-4187-11eb-9721-6826679b7f5a.png">

Reply:
![image](https://user-images.githubusercontent.com/7189118/102672445-17c85380-4191-11eb-8146-47c6c4604ea1.png)

No, it's now all a `call`. `method_call` was removed. Sorry if I was confusing you

Reply:
Ohh, gotcha now. Code updated. Thanks!

Reply:
But unfortunately the updated code, using `call` instead of `method_call` does not produce the expected results:
<img width="421" alt="_HOME_src_ruby-highlight_rb" src="https://user-images.githubusercontent.com/120483/102673295-7be81900-418b-11eb-880b-409dc66430ea.png">

Reply:
Did you `:TSUpdate`? You should on every update of the plugin.

Reply:
![image](https://user-images.githubusercontent.com/7189118/102673513-a4284580-4194-11eb-927d-6ed689c16718.png)

I have those three highlights with the old queries.

Reply:
@theHamsta you're right! `TSUpdate` did the trick. Sorry for the noise.

By the way, how do you show this hover with the highlight names?